source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -145,10 +156,66 @@ corresponding code.
          :end-before: end-change-stream-pipeline
          :language: csharp
 
-To learn more about modifying your change stream output, see the
-:manual:`Modify Change Stream Output
-</changeStreams/#modify-change-stream-output>` section in the {+mdb-server+}
-manual.
+Split Large Change Events Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your application generates change events that exceed 16 MB in size, the
+server returns a ``BSONObjectTooLarge`` error. To avoid this error, you can use
+the ``$changeStreamSplitLargeEvent`` pipeline stage to split the events
+into smaller fragments. The {+driver-short+} aggregation API includes the
+``ChangeStreamSplitLargeEvent()`` method, which you can use to add the 
+``$changeStreamSplitLargeEvent`` stage to the change stream pipeline.
+
+This example instructs the driver to watch for changes and split
+change events that exceed the 16 MB limit. The code prints the
+change document for each event and calls helper methods to
+reassemble any event fragments:
+
```

Comment:
We probably should mention that fragments reassembling is optional (but probably needed), and the split events can be watched as usual events as they arrive.